### DISCUSSION TITLE
Play Store uploading

### DISCUSSION THREAD

**Discussion by BobbyESP** [JunkFood02/Seal#251] [Ideas]
I think that uploading the app to the Play Store would be a nice idea and with that, we are going to catch more users.

**Comment by Marco-9456** [comment]
> I think that uploading the app to the Play Store would be a nice idea and with that, we are going to catch more users.

I don't think it's possible to get it on Play Store because of Google harsh Conditions.

**Reply by BobbyESP** [reply]
Maybe... Because the use of yt-dlp, no?

**Reply by Marco-9456** [reply]
> Maybe... Because the use of yt-dlp, no?

Yes because of yt-dlp

**Comment by BobbyESP** [comment]
Where are the terms for the upload?

**Reply by Marco-9456** [reply]
> Where are the terms for the upload?

https://play.google.com/intl/en-USA/about/developer-content-policy/

**Reply by BobbyESP** [reply]
Thank you!

**Reply by Marco-9456** [reply]
> Thank you!

No problem :)

**Comment by JunkFood02** [comment]
I think Google won't like an app which can download videos from YouTube 🤔

**Reply by BobbyESP** [reply]
Yeah hahaha
